seml/start.py

```diff
@@ -501,19 +600,51 @@ def start_experiments(db_collection_name, local, sacred_id, batch_id, filter_dic
         output_to_file = False
         logging.root.setLevel(logging.VERBOSE)
 
+    if filter_dict is None:
+        filter_dict = {}
+
+    if use_slurm and not set_to_pending:
+        logging.error("use_slurm is True but set_to_pending is False, which are incompatible. Either perform local"
+                      "processing and set set_to_pending to False, or have set_to_pending=True and use_slurm=True.")
+        return
```

Comment:
If the user can't access the set_to_pending option anyway, why do we need this check? This will just be confusing for the user if triggered.

Reply:
Makes sense. See e193786.

Reply:
Yes, you're correct. See d8b23db.